strapi/strapi
# Feat(releases): add release column to CM list view

### What does it do?

Adds the release column to CM list view by injecting column into LV.

### How to test it?

1. On adding entries to releases from CM list view, user should be able to see "to be released in" column with releases added for respective entry. 
2. On clicking releases list, each release should redirect to its release details page.
